download-view/src/main/java/cm/aptoide/pt/download_view/presentation/WalletInstallationViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package cm.aptoide.pt.download_view.presentation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import cm.aptoide.pt.install_manager.InstallManager
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@Suppress("OPT_IN_USAGE")
+@HiltViewModel
+class WalletInstallationViewModel @Inject constructor(
+  installManager: InstallManager
+) : ViewModel() {
+
+  private val appInstaller = installManager.getApp("com.appcoins.wallet")
```

Comment:
maybe use a variable for wallet package name

Reply:
Since this is on another module and is used only here ...